src/app/component/dynamic-elements/dynamic-grid/dynamic-grid.component.ts

```diff
@@ -89,6 +93,10 @@ export class DynamicGridComponent implements OnInit {
   initialization() {
     this.service.getConfiguration(this.path, this.name).subscribe((data) => {
       this.config = data;
+      this.config.paging.settings.pageSize = 1;
+      if(this.savePage['home/reservations']) {
```

Comment:
This is a dynamic network, so it is used in more than one place.
So we need for this some dynamic solution, get the required path from URL or something like that.